src/js/services/auth.js

```diff
@@ -0,0 +1,7 @@
+import { core } from './core';
```

Comment:
Is this only used in more than one place? Does it deserve to be its own service?